pulstar/src/utils/parse_file.rs

```diff
@@ -0,0 +1,347 @@
+use core::f64;
```

Comment:
This whole file can be drastically shortened when the input file is converted to YAML.
The file name `parse_file.rs` does not reveal which file it concerns.